.github/workflows/vmbackup-build.yml

```diff
@@ -0,0 +1,51 @@
+name: vmbackup-build
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - main
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
```

Comment:
Why do you need QEMU for building Docker images?

Reply:
Because we are generating armv7 and arm64 packages too as the base image from vmbackup is having those packages also.